miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditRunController.java

```diff
@@ -214,7 +221,7 @@ public ModelAndView setupForm(Run run, ModelMap model) throws IOException {
         model.put("multiplexed", isMultiplexed(run));
         model.put("metrics",
             getSources().filter(Objects::nonNull).map(source -> source.fetchMetrics(run))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor the code so this stream pipeline is used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3958)